docusaurus/src/pages/about-search.md

```diff
@@ -0,0 +1,10 @@
+
+# How Search Works
```

Comment:
this seems like meta-doc about the site itself. Why add this to the site itself and not just as a markdown page of the repo itself? I'm not sure this needs to be published?

Reply:
Agreed. I moved the Algolia search documentation to the main README.